test/unittest/xmlparser/XMLParserTests.cpp

```diff
@@ -1071,6 +1071,150 @@ TEST_F(XMLParserTests, parseXMLTransportData)
     }
 }
 
+
+/*
+ * This test checks the return of the parseXMLTransportData whitelist argument passed byt name  and the storage of the values in the XMLProfileManager
+ * xml is parsed
+ * 1. Check the correct parsing of a UDP transport descriptor for birth v4 and v6
+ * 2. Check the correct parsing of a TCP transport descriptor for birth v4 and v6
+ */
+TEST_F(XMLParserTests, parseXMLWhiteListByName)
```

Comment:
I think it'd be better to extend `TEST_F(XMLParserTests, parseXMLTransportData)` instead of adding a new test